### ISSUE TITLE
Audio issue: Characters running out of voice lines

### ISSUE THREAD

**Issue by xChillax** [CookiePLMonster/SilentPatchBully#80]
There is a behavior with voice lines that used to work fine, but in SE it broke, which causes characters to always repeat their LAST voice line over and over.

Let's say a character has 4 fighting quotes. It will first play quote 1, then 2, then 3, then 4, and then it will keep playing 4 over and over. since it "ran out of quotes". This will only be fixed if you restart the game. Reloading a save doesn't fix the issue.

How to replicate (2 ways):
1. Fight someone, and stall the fight as long as possible, and then they will eventually run out of quotes and repeat their last quote over and over.
2. Talk/Insult someone constantly until Jimmy runs out of quotes.

This issue is not present in other versions of the game afaik.

I assume the issue is that system goes only forward and never "loops around" once it has no more quotes.

**Comment by xNachito2002** [comment]
Yeah, this same small bug happens to me. If you trigger all NPCs quotes, they'll repeat one quote over and over again. It seems that no one posted a method to fix this...
